components/bl-leafletMap-component/src/svg/position.js

```diff
@@ -0,0 +1,8 @@
+export function PositionSVG() {
```

Comment:
all icons in one file and rename to PositionIcon and so on

Reply:
also, could we initialize it with leaflet here?

Reply:
All icons in the library are in png format, but we don't use png.